activesupport/lib/active_support/deprecation/proxy_wrappers.rb

```diff
@@ -120,7 +120,14 @@ def warn(callstack, called, args)
     #   # => DEPRECATION WARNING: PLANETS is deprecated! Use PLANETS_POST_2006 instead.
     #        (Backtrace information…)
     #        ["Mercury", "Venus", "Earth", "Mars", "Jupiter", "Saturn", "Uranus", "Neptune"]
-    class DeprecatedConstantProxy < DeprecationProxy
+    class DeprecatedConstantProxy < Module
+      def self.new(*args, &block)
```

Comment:
Is this needed?

`initialize`'s first argument is `old_const`, which is a string and should never be nil, so I don't think this method is necessary.

Reply:
Yep, removed (was copied and paste from the `DeprecationProxy` class).